samples/bluetooth/broadcaster_multiple/prj.conf

```diff
@@ -0,0 +1,23 @@
+CONFIG_BT=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
+CONFIG_BT_DEVICE_NAME="Broadcaster Multiple"
+
+CONFIG_BT_DEBUG_LOG=y
+
+# Zephyr Bluetooth LE Controller will need to use chain PDUs when AD data
+# length > 191 bytes
+# - 31 bytes will use 22 bytes for the default name in this sample plus 9 bytes
+#   for manufacturer data
+# - 191 bytes will use 22 bytes for the default name in this sample plus 169
+#   bytes for manufacturer data
+# - 277 bytes will use 22 bytes for the default name in this sample plus 255
+#   bytes for manufacturer data
+# CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=192
+
+# Increase Advertising PDU buffers to number of advertising sets times the
+# number of chain PDUs per advertising set when using Zephyr Bluetooth LE
+# Controller
+# CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+# CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=2
```

Comment:
Do we need all this commented out?

Reply:
Yes, so that host only sample like for qemu_x86 etc. can build in CI. These Kconfigs are covered by the added bsim test in this PR, and hence do not need to  explicitly be build tested by twister here for host+controller combined sample.